src/qibo_cloud_backends/braket_client.py

```diff
@@ -0,0 +1,65 @@
+import time
+
+from braket.aws import AwsDevice
+from braket.devices import LocalSimulator
+from qibo import Circuit as QiboCircuit
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes
+
+from qibo_cloud_backends.braket_translation import to_braket
+
+
+class BraketClientBackend(NumpyBackend):
+    def __init__(self, device=None, verbatim_circuit=False, verbosity=False):
+        """Backend for the remote execution of AWS circuits on the AWS backends.
+
+        Args:
+            device (str): The string representing the ARN of the Braket device. Defaults to Braket's statevector LocalSimulator,
+                          LocalSimulator("default"). Other devices are Braket's density matrix simulator,
+                          LocalSimulator("braket_dm"), or any other QPUs.
+            verbatim_circuit (bool): If `True`, to_braket will wrap the Braket circuit in a verbatim box to run it on the QPU
+                                     without any transpilation. Defaults to `False`.
+            verbosity (bool): If `True`, the status of the executed task will be displayed. Defaults to `False`.
+        """
+        super().__init__()
+
+        self.verbatim_circuit = verbatim_circuit
+        self.verbosity = verbosity
+
+        self.device = AwsDevice(device) if device else LocalSimulator()
```

Comment:
I feel that it will only be fair for the user if we have the LocalSimulator around despite being a cloud provider. Any execution of circuits on the Braket devices will incur a cost, even if it's just 1 shot. So having the LocalSimulator around can allow the user to verify that their code is executing properly before letting it be executed on the Braket device.

Reply:
That's definitely correct. So, it's definitely meaningful to keep, at least for validation.
I wonder if you could even validate the circuit correctness without executing, even if a fully invalid circuit should not be billed, I guess...

In any case, we could even close this comment at this point.

Reply:
Thanks @alecandido! We can definitely find a way to validate a circuit's correctness but perhaps through opening an issue?

Reply:
Well, if it's not going to cost you money, maybe it's not even worth to validate. We could save the issue, for the time being.